src/Uno.UI/UI/Xaml/UIElement.TextHelper.wasm.cs

```diff
@@ -43,7 +43,7 @@ internal void SetText(string text)
 
 		internal void SetFontStyle(object localValue)
 		{
-			if (localValue is UnsetValue)
+			if (localValue == DependencyProperty.UnsetValue)
```

Comment:
![image](https://github.com/unoplatform/uno/assets/31348972/095afe37-5967-4e1d-b7d4-46fbc7b9c8b2)

Same with the other changes in the same file.